src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -173,7 +171,7 @@ public override void OnStopActivity(Activity activity, object payload)
                 else
                 {
                     Status status = SpanHelper.ResolveSpanStatusForHttpStatusCode(response.StatusCode);
-                    activity.SetStatus(status.WithDescription(response.HttpContext.Features.Get<IHttpResponseFeature>()?.ReasonPhrase));
+                    activity.SetStatus(status);
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#status
_"Don't set the span status description if the reason can be inferred from http.status_code."_